rpc/namespaces/ethereum/eth/filters/api.go

```diff
@@ -353,6 +355,8 @@ func (api *PublicFilterAPI) NewHeads(ctx context.Context) (*rpc.Subscription, er
 		return &rpc.Subscription{}, err
 	}
 
+	// TODO: use events
```

Comment:
Can we track this TODO somewhere?
What does it implies not using event currently, what benefices we'll get from events in the future?